src/main/java/co/omise/models/ScopedList.java

```diff
@@ -95,10 +96,10 @@ public Options order(Ordering order) {
                 map.put("limit", limit.toString());
             }
             if (from != null) {
-                map.put("from", formatter.print(from));
+                map.put("from", from.atZone(ZoneId.of("Z")).format(formatter));
```

Comment:
Why do we need to convert an Instant to ZonedDateTime before doing the formatting here?

Reply:
I think `formatter.format(from)` should work here